ayan-biswas0412.md

```diff
@@ -0,0 +1,49 @@
+---
+name: "@ayan-biswas0412"
+project: "Camera Based Door Access Control System"
+---
+
+# Camera Based Door Access Control System
+
+## Summary
+
+I am going to build a camera based door access control system that uses facial recognition to grant or deny access to a locked door. 
+The system will include a camera mounted near the door, a small computer for processing the video feed and running the facial recognition algorithm, and a lock mechanism that can be controlled by the computer. 
+I am excited to build this because I think it has the potential to improve security and convenience in a variety of settings, such as office buildings, apartment complexes, and homes.
+
+## Plan
+
+1. Gather materials and tools needed:
+    * Raspberry Pi microcomputer
+    * Raspberry Pi camera module
+    * Servo motor
+    * Lock mechanism (such as a deadbolt or latch bolt)
+    * Wires and connectors
+    * Power supply
+2. Set up the Raspberry Pi and camera module according to the manufacturer's instructions.
+3. Connect the servo motor to the Raspberry Pi and wire it to control the lock mechanism.
+4. Write and upload code to the Raspberry Pi that will:
+    * Continuously capture video from the camera
+    * Run the facial recognition algorithm on the video feed
+    * Grant or deny access to the door based on the result of the facial recognition
+5.Test the system to ensure it is functioning properly.
+6. Install the camera, servo motor, and lock mechanism near the desired door.
+
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Raspberry Pi 3 - Model B - ARMv8 with 1G RAM   | [https://www.adafruit.com/product/3055](https://www.adafruit.com/product/3055) | $35  |
+| Raspberry Pi Camera Board v2 - 8 Megapixels   | [https://www.adafruit.com/product/3099](https://www.adafruit.com/product/3099) | $29.95  |
+| Micro servo   | [https://www.adafruit.com/product/169](https://www.adafruit.com/product/169) | $5.95  |
+| Rotating Door Lock DC Motor| [https://www.adafruit.com/product/3881](https://www.adafruit.com/product/3881) | $2.50  |
+| Wires and connectors  | [https://www.adafruit.com/product/1957](https://www.adafruit.com/product/1957) | $1.95  |
+| 5V 2.5A Switching Power Supply with 20AWG MicroUSB Cable   | [https://www.adafruit.com/product/1995](https://www.adafruit.com/product/1995) | $8.25  |
+| HDMI 5" 800x480 Display Backpack - With Resistive Touchscreen   | [https://www.adafruit.com/product/2260](https://www.adafruit.com/product/2260) | $74.95  |
+| Speaker   | [https://www.adafruit.com/product/3885](https://www.adafruit.com/product/3885) | $5.95  |
```

Comment:
what's the role of the speaker?

Reply:
It is for making a sound like -> 
Access Granted/Denied for the user

Reply:
ahh cool!!